### PR TITLE
Update readme for ng9 support

### LINKED ISSUE #64: NG9 Support
#### Describe the bug
Unable to render SVG icons following current instructions in a NG9 project.

#### What is the expected behavior?
Instructions for how to render our icons should work for NG9 and older versions.

#### What are the steps to reproduce?
1. Create a new angular project using pxb-cli.  
2. Following the instructions on how to integrate svg pxblue icons.
3. Observe that icon will not render and error occurs in console.

#### Screenshots or links to minimum reproduction example
image

#### Environment

Angular 9, version 1.0.2 of pxb-cli and latest version of icons package.

### PR DESCRIPTION
**Pull request by daileytj**
Fixes #64.

#### Changes proposed in this Pull Request:
- Update readme for ng9 support

### CODE REVIEW

**Comment by joebochill** [comment]
Nice! Is this all that was required to get it to work?

**Comment by daileytj** [comment]
> Nice! Is this all that was required to get it to work?

Yep! Not entirely sure what the cause is for this but here's the thread I found the fix in: https://github.com/angular/components/issues/20274

File: svg/README.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ export class AppComponent {
   ) {
     this.matIconRegistry.addSvgIcon(
       "breaker",
-      this.domSanitizer.bypassSecurityTrustResourceUrl(icon)
+      this.domSanitizer.bypassSecurityTrustResourceUrl(icon.default)
```
Comment (joebochill) [inline svg/README.md:62]: Do we want to put the `.default` here, or would it be better in the import?

```
const iconSet = require(......).default;
```

Comment (daileytj) [reply svg/README.md:62]: No preference on my end. Might be a little more clear if its on the import though. I'll change it up

**Review by joebochill — APPROVED** [review]

**Pull request merged**